src/pages/Defi/components/EligibleCarousel.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@chakra-ui/react'
+import { bnOrZero } from '@shapeshiftoss/investor-foxy'
+import { useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { NavLink } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Carousel } from 'components/Carousel/Carousel'
+import { bn } from 'lib/bignumber/bignumber'
+import { selectAggregatedEarnUserStakingEligibleOpportunities } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FeaturedCard } from './FeaturedCards/FeaturedCard'
+
+export const EligibleCarousel = () => {
+  const translate = useTranslate()
+  const eligibleOpportunities = useAppSelector(selectAggregatedEarnUserStakingEligibleOpportunities)
+  const filteredEligibleOpportunities = useMemo(() => {
+    return eligibleOpportunities
+      .filter(o => bnOrZero(o.tvl).gt(50000))
+      .sort((a, b) => bn(b.apy).toNumber() - bn(a.apy).toNumber())
```

Comment:
Only `toNumber` the output to satisfy `Array.prototype.sort()`, floats comparison should be done with BigNumber.JS

```suggestion
      .sort((a, b) => bn(b.apy).minus(a.apy).toNumber()
```